tests/terraform/checks/resource/gcp/example_DataprocPrivateCluster/main.tf

```diff
@@ -0,0 +1,64 @@
+################
+## PASS TESTS ##
+################
+
+resource "google_dataproc_cluster_iam_binding" "pass1" {
+  cluster = "my-private-cluster-binding1"
+  role    = "roles/dataproc.serviceAgent"
+  members = [
+    "user:jane@example.com",
+  ]
+}
+
+resource "google_dataproc_cluster_iam_binding" "pass2" {
+  cluster = "my-private-cluster-binding2"
+  role    = "roles/dataproc.viewer"
+  members = [
+    "user:jason@example.com",
+  ]
+}
+
+resource "google_dataproc_cluster_iam_member" "pass1" {
+  cluster = "my-private-cluster-member1"
+  role    = "roles/dataproc.worker"
+  member  = "group:mygroup@example.com"
+}
+
+resource "google_dataproc_cluster_iam_member" "pass2" {
+  cluster = "my-private-cluster-member2"
+  role    = "roles/dataproc.editor"
+  member  = "domain:example.com"
+}
+
+
+################
+## FAIL TESTS ##
+################
+
+resource "google_dataproc_cluster_iam_binding" "fail1" {
+  cluster = "my-public-cluster-binding1"
+  role    = "roles/dataproc.hubAgent"
+  members = [
```

Comment:
a check with a list containing a good member and one bad member would be great

Reply:
Great idea! Added via https://github.com/bridgecrewio/checkov/pull/2450/commits/fdcfc1a7253e0286c415563495568aced0b1e674